lang/llvm-3.4/Portfile

```diff
@@ -77,15 +75,12 @@ if {${subport} eq "llvm-${llvm_version}"} {
 }
 
 if {${os.platform} eq "darwin" && ${os.major} < 11 && ${cxx_stdlib} eq "libc++"} {
-    configure.cxx_stdlib    libstdc++
+    revision                    [ expr ${revision} + 1 ]
+    configure.cxx_stdlib        libstdc++
     # Have to also use bootstrap versions of deps that use libstdc++ in
     # order to be able to build libc++.
-    depends_lib-replace         port:python27 port:python27-bootstrap \
-                                port:ncurses port:ncurses-bootstrap
-
-    configure.cppflags-prepend  -I${bootstrap_prefix}/include
-    configure.ldflags-prepend   -L${bootstrap_prefix}/lib
-    configure.env-append        PATH=${bootstrap_prefix}/bin:$::env(PATH)
```

Comment:
I assume you checked that this works right without the PATH change?

Reply:
@jmroot yes, I've build it it.